packages/forms/frontend/sirius-components-widget-reference/src/components/ValuedReferenceAutocomplete.tsx

```diff
@@ -140,10 +150,11 @@ export const ValuedReferenceAutocomplete = ({
           placeholder={placeholder}
           InputProps={{
             ...params.InputProps,
+            style: { paddingRight: theme.spacing(10) },
```

Comment:
Could you also put this style into the `useStyles` please ? (the same way as the `InputProps` in `ModelBrowserFilterBar`)

Reply:
It's the `Autocomplete` that customize the textfield style by passing `className` with `...params.InputProps`, if we override it by making our own style we lost some wanted behavior. 
Maybe I missed something but I couldn't do anything other than add style this way.

Reply:
ok